submissions/SerhiiYakovenko/html-css-popup/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>CyberPunk</title>
+  <link href="./css/style.css" rel="stylesheet" />
+  <link rel="icon" href="./img/favicon.png" type="image/png" />
+  <link href="https://fonts.googleapis.com/css2?family=Orbitron&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+  <h1>
```

Comment:
I see that the first focusable element by keyboard now is this link, but navigation elements should be the first focusable elements. So I suggest moving this title with the link after the `nav` element. It will help to solve this problem.